lib/src/otx/data/entity/sample.py

```diff
@@ -116,3 +117,20 @@ def from_dm_item(cls, item: DatasetItem) -> ClassificationSample:
         )
         sample.img_info = img_info
         return sample
+
+class SegmentationSample(OTXSample):
+    """OTXDataItemSample is a base class for OTX data items."""
+
+    image: np.ndarray | tv_tensors.Image = image_field(dtype=pl.UInt8)
+    masks: np.ndarray | tv_tensors.Mask = mask_field(dtype=pl.UInt8)
+    dm_image_info: DmImageInfo = image_info_field()
+    
+    def __post_init__(self) -> None:
+        shape = (self.dm_image_info.height, self.dm_image_info.width)
+        self.image = tv_tensors.Image(self.image.transpose(2, 0, 1))
+        self.masks = tv_tensors.Mask(self.masks[np.newaxis, ...])
```

Comment:
I have created tickets to avoid those steps by adding native support for tv_tensors and channel first in Datumaro: https://github.com/open-edge-platform/datumaro/issues/1872, https://github.com/open-edge-platform/datumaro/issues/1873